e2e/app/create-account-page.ts

```diff
@@ -154,7 +154,7 @@ export default class CreateAccountPage extends BasePage {
     const emailAddress = await Textbox.forLabel(this.page, {textContains: FIELD_LABEL.INSTITUTION_EMAIL, ancestorNodeLevel: 2});
```

Comment:
I would pass an institution input object into this method so that you can fill it out more than one way to test edge cases, validation, etc, if you want.

Reply:
I can refactor in a new PR. This PR only deals with one test failure that only happens on CircleCi.